assets/src/blocks/Carouselheader/CarouselHeaderBlock.js

```diff
@@ -256,9 +268,7 @@ export class CarouselHeaderBlock {
           removeSlide={removeSlide}
         />;
       },
-      save() {
-        return null;
-      }
+      save: frontendRendered( BLOCK_NAME )
```

Comment:
Hey @dpivo this is preventing the Carousel from rendering in the frontend, as it wasn't converted to WYSIWYG yet. 

Maybe the ticket needs refinement, blocks returning `null` are not expected to have any HTML representation in the Editor. I think the goal is to fix this:

https://jira.greenpeace.org/browse/PLANET-5463

But the "Edit HTML" option only makes sense for purely static blocks. I think we need to review what is expected and what is plausible. 

cc @comzeradd

Reply:
We could wait till we transform this block too into WYSIWYG